src/mtype.cpp

```diff
@@ -52,6 +57,23 @@ std::string mtype::nname( unsigned int quantity ) const
     return ngettext( name.c_str(), name_plural.c_str(), quantity );
 }
 
+m_size mtype::volume_to_size( const units::volume vol ) const
+{
+    if( vol > 0_ml && vol <= 7500_ml ) {
+        return MS_TINY;
+    } else if( vol > 7500_ml && vol <= 46250_ml ) {
```

Comment:
You don't need the first check, write it like this instead:
```C++
if( vol <= 7500_ml ) {
    return ...;
} else if( vol <= 46250_ml ) {
    return ...;
} else if( vol <= ...
```
The second if clause will only be reached if `vol > 7500_ml`, which does not need to be cecked again.

Reply:
I just hope there will be no situation where `vol == 0` then...